clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java

```diff
@@ -49,7 +47,6 @@ public interface Constants {
     int SESSION_LENGTH_MINS = 20;
     String DEVICE_ID_TAG = "deviceId";
     String FALLBACK_ID_TAG = "fallbackId";
-    SimpleDateFormat FB_DOB_DATE_FORMAT = new SimpleDateFormat("MM/dd/yyyy", Locale.US);
```

Comment:
@piyush-kukadiya this variable was unused?

Reply:
@darshanclevertap YES. It's Facebook profile DOB related code which we removed in the past when removing google and facebook login support

Reply:
Ok thanks!